pkg/servicetest/testdata/services/docker-compose.memcached.yaml

```diff
@@ -0,0 +1,30 @@
+# ddev memcached recipe file
+#
+# To use this in your own project: Copy this file to your project's .ddev folder.
+# Edit the 'command' settings to change CLI flags sent to memcached.
+# Defaults to '-m 128' to allocate 128 MB of memcached storage.
+
+version: '3'
+
+services:
+  memcached: # This is the service name used when running ddev commands accepting the --service flag
+    container_name: ddev-${DDEV_SITENAME}-memcached # This is the name of the container. It is recommended to follow the same name convention used in the main docker-compose.yml file.
+    image: memcached:1.5
+    restart: always
+    ports:
+      - 11211 # memcached is available at this port inside the container
+    labels:
+    # These labels ensure this service is discoverable by ddev
+      com.ddev.site-name: ${DDEV_SITENAME}
+      com.ddev.approot: $DDEV_APPROOT
+      com.ddev.app-url: $DDEV_URL
+    environment:
```

Comment:
I'm pretty sure the entire environment stanza is not useful; we don't support access via http on named project via port 11211 because 11211 is a tcp interface, not http. So ddev-router is not involved at all.  (I tested and it works fine without this section)